packages/map-popup/src/EndpointPopup.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useCallback, useMemo } from "react";
+import { Popup } from "@opentripplanner/base-map";
+import { FormattedMessage } from "react-intl";
+import styled from "styled-components";
+import { Location } from "@opentripplanner/types";
+import { getEntryRelativeTo } from "./util";
```

Comment:
Sort imports... I think in this library it is by package name then folder names?